S23/pbutiya/CodeReview.py

```diff
@@ -0,0 +1,118 @@
+print("")
+print("**************************************************************WELCOME TO THE UTILITY*********************************************")
+print("****************************************************Convert UPPERCASE Into lowercase*********************************************")
+print("*********************************************************************************************************************************")
+print("")
+# Creating an array for Langauge List, In future more langauges can get added to array.
+languageList = ["ENGLISH", "TURKISH or AZERBAIJANI", "IRISH","GREEK"]
+
+# Function to select from the List
+def language_Function():
+    print("")
+    print("****************************************************************PLEASE SELECT LANGAUGE **********************************")
+    print("********* 1."+languageList[0])
+    print("********* 2."+languageList[1])
+    print("********* 3."+languageList[2])
+    print("********* 4."+languageList[3])
+    print("")
+    print("************************************************ NOTE: USE THE SERIAL NUMBER TO SELECT A LANGAUGE************************")
+    print("-------------------------------------------------For example : Press 1 for English---------------------------------------")
+    print("")
+    select_Function()
+    
+# Logic for each langauge, In future more langauges can be added and their logic can be written.
+def select_Function():
```

Comment:
I don't really like the design of this code — you've mixed the logic for lowercasing with a bunch of user-interface menu code.  This is not "modular" is the positive sense we've discussed in class. Could you refactor what you have into a single Python function that takes two strings as arguments (a string to lowercase and a BCP-47 code)?  See the homework specification for more details.